docs/HTTP2.md

```diff
@@ -2,9 +2,9 @@
 
 ## HTTP2
 
-_Fastify_ offers **experimental support** for HTTP2 starting from Node
-8.8.0, which includes HTTP2 without a flag. _Fastify_ supports HTTP2
-both over HTTPS or over plaintext. Note that HTTP2 is available only for node versions >= `8.8.1`.
+_Fastify_ offers **experimental support** for HTTP2 starting from 
+Node 8 LTS (8.9.0), which includes HTTP2 without a flag. _Fastify_ supports HTTP2
+both over HTTPS or over plaintext. Note that HTTP2 is available only for node versions >= `8.9.0`.
```

Comment:
> Note that HTTP2 is available only for node versions >= `8.9.0`.

For me it looks redundand because node >= 8.16.0 required

Reply:
or maybe it would be better to align here to 8.16.0, what do you think ?

Reply:
I think `^8.16.0` would be great

Reply:
Fastify gonna drop node < 8.16.0 support and HTTP/2 available since 8.8.1 hence we can drop out this note

Reply:
done, hope is good enough (and simpler) now ...